database/sql/src/main/scala/cromwell/database/slick/tables/MetadataEntryComponent.scala

```diff
@@ -78,6 +78,21 @@ trait MetadataEntryComponent {
     }
   )
 
+  val metadataTotalSizeRowsForRootWorkflowId = Compiled(
```

Comment:
This query seems almost identical to `metadataEntriesWithoutLabelsForRootWorkflowId`.

Isn't the calculation in `metadataEntriesWithoutLabelsForRootWorkflowId` actually a better estimate because we do not include the labels in the carbonite payload?

Does it make sense to do this instead?
```
val metadataTotalSizeRowsForRootWorkflowId = metadataEntriesWithoutLabelsForRootWorkflowId.size
```

If the premise about labels not mattering for carboniting is false, ignore this comment.

Reply:
`MetadataBuilderActor` tries to read all metadata, including labels, from the DB, so looks like we need to assess size with labels included.